### PR TITLE
Add Brain Age: Concentration Training

### PR DESCRIPTION
**Pull request by RoySullivan**
Adds Brain Age: Concentration Training.

*Automatically generated by PR builder*

### CODE REVIEW

File: games/brain-age-concentration-training/game.dat [hunk 1]
```diff
@@ -0,0 +1,9 @@
+title = "Brain Age: Concentration Training"
+description = "Brain Age: Concentration Training is a puzzle video game, it's the forth installment of the series and exclusively made for the Nintendo 3DS. it tests your 'brain age' based on how well you perform exercises like but not limited to addition, subtraction, memory and more."
```
Comment (Leo626) [inline games/brain-age-concentration-training/game.dat]: Use this description from Wikipedia with some tweaks.

> Brain Age: Concentration Training, known in Europe and Australia as \"Dr. Kawashima's Devilish Brain Training: Can You Stay Focused?\", is a puzzle video game developed and published by Nintendo, and is the fourth major entry in the Brain Age series. It features a selection of activities and minigames that are designed to stimulate and improve the player's concentration and working memory.

File: games/brain-age-concentration-training/game.dat [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+title = "0004000000043700"
+region = "jpn"
+release_date = "2012-07-28"
+
+[[releases]]
+title = "00040000000CB100"
+region = "kor"
+release_date = "2013-09-05"
+
+[[ testcases ]]
```
Comment (Leo626) [inline games/brain-age-concentration-training/game.dat]: Also testcases aren't added here anymore. You submit them through Citra (About -> Report Compatibility).

**Comment by Leo626** [comment]
Are `Main-Menu-Devilish-Training.png` and `title-screen.png` duplicates?

**Comment by RoySullivan** [comment]
i'm new to this, should i do anything else?

**Comment by Leo626** [comment]
Nothing else for now. Just need to wait for another person to review and approve before it's merged.

File: games/brain-age-concentration-training/game.dat [hunk 3]
```diff
@@ -0,0 +1,24 @@
+title = "Brain Age: Concentration Training"
+description = "Brain Age: Concentration Training, known in Europe and Australia as \"Dr. Kawashima's Devilish Brain Training: Can You Stay Focused?\", is a puzzle video game developed and published by Nintendo, and is the fourth major entry in the Brain Age series. It features a selection of activities and minigames that are designed to stimulate and improve the player's concentration and working memory."
```
Comment (Hexagon12) [inline games/brain-age-concentration-training/game.dat]: The slashes in the quotes are not needed

Comment (Leo626) [reply games/brain-age-concentration-training/game.dat]: They are needed else Travis will fail.

**Comment by RoySullivan** [comment]
Wiki name fixed!

**Pull request merged**